src/globus_sdk/experimental/globus_app/app.py

```diff
@@ -224,8 +224,67 @@ def _initialize_authorizer_factory(self) -> None:
         authorize requests.
         """
 
+    def login(
+        self,
+        *,
+        auth_params: GlobusAuthorizationParameters | None = None,
+        force: bool = False,
+    ) -> None:
+        """
+        Log an auth entity into the app, if needed, storing the resulting tokens.
+
+        A login flow will be is run if:
+            * `auth_params` are provided
+            * `force` is set to True
+            * `self.login_required()` evaluates to True
```

Comment:
```suggestion
            * ``auth_params`` are provided,
            * ``force`` is set to True, or
            * ``self.login_required()`` evaluates to True.
```

Reply:
Looks like part of this change is missing? I can check others, but might be a little bit before I can swing back through.

Reply:
Oh my mistake. I didn't see the list-form part of the suggestion here, just the "``" to replace "`" enveloping.

I can update to that style as well.

Reply:
Updated in latest rev.